test/shelf_io_test.dart

```diff
@@ -43,7 +43,7 @@ void main() {
   });
 
   test('sync null response leads to a 500', () async {
-    await _scheduleServer((request) => null);
+    await _scheduleServer((request) => Response.internalServerError());
```

Comment:
This changed the intent of the test. It no longer matches the name.

Should it be removed? Should it be moved to an `opt_out_test.dart`?

Reply:
Good note! Hrm....